widgets/materialdesign.md

```diff
@@ -4,4 +4,4 @@ title: Material Components Widgets
 
 permalink: widgets/material/
 ---
-{% include catalogpage.html category="Material Design"%}     
```

Comment:
I'm new here. Is this creating the category or referring to one made elsewhere?

Reply:
Good question! maybe refers to https://github.com/flutter/website/blob/master/_data/catalog/index.json ?

Reply:
I'm new here too: I figure it is referring to the catalog index: https://github.com/flutter/website/pull/805/files#diff-5987947d3f8772d6741c18d0546ab19eL9, where the 'Material Design' entry was renamed to 'Material Components' necessitating the link update here as well.

Reply:
Then this is really a fix! Thank you.